jupyterhub_files/spawner.py

```diff
@@ -36,6 +36,9 @@ def get_local_ip_address():
     {"Key": "Owner", "Value": SERVER_PARAMS["WORKER_SERVER_OWNER"]},
     {"Key": "Creator", "Value": SERVER_PARAMS["WORKER_SERVER_OWNER"]},
     {"Key": "Jupyter Cluster", "Value": SERVER_PARAMS["JUPYTER_CLUSTER"]},
+    {"Key": "environment", "Value": "production"},
```

Comment:
I think it might be useful to make this an optional server parameter that defaults to _production_. That would allow us to set it for development/sandbox environments too.